samples/NativeLibrary/README.md

```diff
@@ -1,56 +1,181 @@
+
```

Comment:
Could you please delete these extra whitespace so that this document matches the style used everywhere else in the repo?